scripts/joinmarket-qt.py

```diff
@@ -2005,13 +2006,19 @@ def loadWalletFromBlockchain(self, firstarg=None, pwd=None):
         self.wallet_service.autofreeze_warning_cb = self.autofreeze_warning_cb
         self.wallet_service.startService()
         self.syncmsg = ""
-        self.walletRefresh = task.LoopingCall(self.updateWalletInfo)
+        self.walletRefresh = task.LoopingCall(self.updateWalletInfo, None, None)
```

Comment:
If we only need to call `updateWalletInfo` via `self.walletRefresh` once, it feels like a `LoopingCall` isn't the best candidate to schedule such a non-repeating task.

Reply:
So partially because the name is no longer appropriate, it may be unclear that the purpose of that looping call is now only to wait for the wallet sync to finish (which can take a while, although typically the loop will only fire once). The loop could be renamed to `sync_wait_loop` or similar to remove confusion.
Once sync completes the loop stops.
(Also, this could be done a bit better with a deferred, but the difference is minor and it'd be fine to have someone else PR that later if they want).